devtools/maven/src/main/java/io/quarkus/maven/UpdateMojo.java

```diff
@@ -59,11 +59,11 @@ public class UpdateMojo extends QuarkusProjectStateMojoBase {
     private Boolean rewriteDryRun;
 
     /**
-     * The io.quarkus:quarkus-update-recipes version. This artifact contains the base recipes used by this tool to update a
-     * project.
+     * Use a custom quarkus-update-recipes coords (GAV) or just version. This artifact contains the base recipes used by this
+     * tool to update a project.
      */
-    @Parameter(property = "updateRecipesVersion", required = false)
```

Comment:
this will break existing users by removing the old option. any reason why we don't support the old one as deprecated alternative?

Reply:
This is a very advanced feature, adding backward compatibility needs to be done on all tools (CLI + Maven + Gradle). I think we are fine renaming it.

Reply:
The only case to use it was to develop and try change in the quarkus recipes. Now this might be used to customise it, so we won't change it anymore.

Reply:
if we are breaking backward compatability we should at least ensure what replaces it is consistent.

so the idea is to have `updateRecipeCoords` and then `additionalUpdateRecipeCoords` ?

one being singular the other a list?

in both cases I think we should drop `Coords` as that is not something we use elsewhere (afaik).
i.e. we don't use `extensionCoords` where we refer to extensions.

And I suggest it becomes:

`updateRecipe` and `additionalUpdateRecipes` to signal one is singular the other plural/list.

Reply:
do I need to go ahead and update https://github.com/quarkusio/quarkus/pull/41663 with `additionalUpdateRecipes`  (I changed it multiple times so ideally we all agree on what it should be). 
do we need both PRs? if so, in what order?

Reply:
ah. I see you merged it already @ia3andy